sdk/search/Azure.Search.Documents/src/Models/SearchResults.cs

```diff
@@ -53,6 +53,10 @@ public class SearchResults<T>
         /// </summary>
         public IDictionary<string, IList<FacetResult>> Facets { get; internal set; }
 
+        /// <summary> The answers query results for the search operation;
+        /// <c>null</c> if the <see cref="SearchOptions.QueryAnswer"/> parameter was not specified or set to <see cref="QueryAnswer.None"/>. </summary>
+        public IDictionary<string, IList<AnswerResult>> Answers { get; internal set; }
```

Comment:
I am working on it. Last I checked, this was also returned as an array. Now, I am not getting this in the response at all.